### ISSUE TITLE
[RUST_SHORT]: Remove requirement of using cargo workspaces?

### ISSUE THREAD

**Issue by ifaoji** [42-Short/shortinette#195]
Should we remove the requirement to use cargo workspaces?

The requirement of using cargo workspace jumps in a bit random and imo is also not really necessary to learn Rust. It just adds an additional barrier which is not really needed.

And also in the 42 C/C++ World we do not use anything similar to that.

**Comment by winstonallo** [comment]
I agree, it was just confusing and an additional failing reason that didn't add any educational value
